tbdex/offering/offering.go

```diff
@@ -29,6 +29,13 @@ type Data struct {
 	Payin          *PayinDetails                 `json:"payin,omitempty"`
 	Payout         *PayoutDetails                `json:"payout,omitempty"`
 	RequiredClaims *pexv2.PresentationDefinition `json:"requiredClaims,omitempty"`
+	Cancellation   *Cancellation                 `json:"cancellation,omitempty"`
```

Comment:
not sure `omitempty` does anything meaningful here but from the spec this is a required data property

Reply:
i think omitempty is used to exclude the field when serializing to json